README.md

```diff
@@ -9,16 +14,22 @@ The easiest way to deploy **dashboard** is using *Docker*.
 docker run -p 8080:80 bblfsh/dashboard -bblfsh-addr <bblfsh-server-addr>
 ```
 
-If you don't have a bblfsh server running you can execute the dashboard and the server using the following command:
+If you don't have a bblfsh server running, please read the [getting started](https://doc.bblf.sh/user/getting-started.html) guide, to learn more about how to use and deploy a bblfsh server.
+
+If don't want to run **dashboard** using our *Docker* image you can download a binary from [releases](https://github.com/bblfsh/dashboard/releases) page and run it as
 
 ```sh
-docker run --privileged -d -p 9432:9432 --name bblfsh bblfsh/server
-docker run -p 8080:80 --link bblfsh bblfsh/dashboard -bblfsh-addr bblfsh:9432
+./bblfsh-dashboard -bblfsh-addr <bblfsh-server-addr>
 ```
 
-Please read the [getting started](https://doc.bblf.sh/user/getting-started.html) guide, to learn more about how to use and deploy a bblfsh server.
+## Architecture
```

Comment:
This is not relevant for user, only for contributors, remove from here and place in a `CONTRIBUTING.md` file.